blockchain/notifications.go

```diff
@@ -128,9 +128,8 @@ type BlockConnectedNtfnsData struct {
 	// chain.
 	ParentBlock *dcrutil.Block
 
-	// IsTreasuryActive indicates whether or not the treasury agenda is active
-	// for the block that was connected.
-	IsTreasuryActive bool
+	// Flags indicates which agendas are active for the block that was connected.
```

Comment:
One thing that does slightly concern me is that by including the flags in the connect/disconnect notifications, they sort of implicitly morph into something more than just existing for checking transactions, which is what they currently are used for and documented accordingly.  While that isn't necessarily a big deal on its own, it does carry a new implication that all active agendas (for which there are flags) will now be set, which means care will need to be taken when sending those notifications to ensure they're all set.

I wonder if this (and the other case in the disconnect notification) should specifically call out that these flags are not necessarily for all agendas, rather only the agendas which should be considered as active when checking transactions?

Reply:
That's a good point.  I can see how that might not be clear without looking at the calling code.  I updated it to include a comment that indicates that it represents the agendas _for checking transactions_.  I also update the naming of the fields/variables to `CheckTxFlags`/`checkTxFlags` since it makes it a bit more clear and is the naming you had already used in one other location.  Let me know if that seems a bit more clear to you.

Reply:
Nice.  I think those updates make it nice and clear.